lms/envs/common.py

```diff
@@ -2870,3 +2870,14 @@
 TYPES_WITH_CHILD_PROBLEMS_LIST = [
     'library_content',
 ]
+
+# These are the problem types (by block_type) that an instructor can query over in the Instructor Query tab
+INSTRUCTOR_QUERY_PROBLEM_TYPES = [
```

Comment:
@sigberto We're working on standardizing that, but in the interim, just stick it on the bottom, I guess? Or better yet, maybe near `TYPES_WITH_CHILD_PROBLEMS_LIST` if it's there? (or something similar)?